create_package.sh

```diff
@@ -26,7 +26,7 @@ cp /ovms/src/custom_nodes/tokenizer/build/src/lib*tokenizer.so /ovms_release/lib
 if [ -f /openvino_contrib/modules/custom_operations/user_ie_extensions/user_ie_extensions/libuser_ov_extensions.so ]; then cp -v /openvino_contrib/modules/custom_operations/user_ie_extensions/user_ie_extensions/libuser_ov_extensions.so /ovms_release/lib/ ; fi
 if [ -f /openvino/bin/intel64/Release/libopenvino_nvidia_gpu_plugin.so ]; then cp -v /openvino/bin/intel64/Release/libopenvino_nvidia_gpu_plugin.so /ovms_release/lib/ ; fi
 
-find /ovms/bazel-out/k8-*/bin -iname '*.so*' ! -type d ! -name "*params" -exec cp -v {} /ovms_release/lib/ \;
+find /ovms/bazel-out/k8-*/bin -iname '*.so*' ! -type d ! -name "*params" ! -name "lib_node_*" -exec cp -v {} /ovms_release/lib/ \;
```

Comment:
It took me a moment to make sure those are test custom nodes. Maybe we could rename them to:
**"lib_testnode_*"** to avoid confusion at some point in the future?